From a3bffc30602ce3f87d2b78ad8f8c449ef6ec0647 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Wed, 20 Nov 2019 06:57:44 -0600
Subject: [PATCH 02/34] ARM64: dts: lpd-imx8mm-evk: Remove unused items

The original branch of the DTS file had many remnants of the
original NXP/Freescale devkit that were unused.

This patch removes the unused items from the device tree.
---
 .../boot/dts/freescale/lpd-imx8mm-evk.dts     | 329 +-----------------
 1 file changed, 1 insertion(+), 328 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/lpd-imx8mm-evk.dts b/arch/arm64/boot/dts/freescale/lpd-imx8mm-evk.dts
index 359fdbc07684..667b71fdbb85 100644
--- a/arch/arm64/boot/dts/freescale/lpd-imx8mm-evk.dts
+++ b/arch/arm64/boot/dts/freescale/lpd-imx8mm-evk.dts
@@ -73,15 +73,7 @@
 			off-on-delay-us = <12000>;
 		};
 
-		/*rfee reg_usdhc2_vmmc: regulator-usdhc2 {
-			compatible = "regulator-fixed";
-			regulator-name = "VSD_3V3";
-			regulator-min-microvolt = <3300000>;
-			regulator-max-microvolt = <3300000>;
-			gpio = <&gpio2 19 GPIO_ACTIVE_HIGH>;
-			off-on-delay = <20000>;
-			enable-active-high;
-		};*/
+
 		
 		/* regulator for usb otg */
 		reg_usb_otg_vbus: usb_otg_vbus {
@@ -122,63 +114,9 @@
 		clock-names = "mclk";
 		wlf,mute-gpios = <&gpio5 21 GPIO_ACTIVE_LOW>;
 	};
-/* rfee 
-	sound-wm8524 {
-		compatible = "fsl,imx-audio-wm8524";
-		model = "wm8524-audio";
-		audio-cpu = <&sai3>;
-		audio-codec = <&wm8524>;
-		audio-routing =
-			"Line Out Jack", "LINEVOUTL",
-			"Line Out Jack", "LINEVOUTR";
-	};
-
-	sound-ak4458 {
-		compatible = "fsl,imx-audio-ak4458";
-		model = "ak4458-audio";
-		audio-cpu = <&sai1>;
-		audio-codec = <&ak4458_1>, <&ak4458_2>;
-		ak4458,pdn-gpio = <&pca6416 4 GPIO_ACTIVE_HIGH>;
-	};
-
-	sound-ak5558 {
-		compatible = "fsl,imx-audio-ak5558";
-		model = "ak5558-audio";
-		audio-cpu = <&sai5>;
-		audio-codec = <&ak5558>;
-		status = "disabled";
-	};
-
-	sound-ak4497 {
-		compatible = "fsl,imx-audio-ak4497";
-		model = "ak4497-audio";
-		audio-cpu = <&sai1>;
-		audio-codec = <&ak4497>;
-		status = "disabled";
-	};
-
-	sound-spdif {
-		compatible = "fsl,imx-audio-spdif";
-		model = "imx-spdif";
-		spdif-controller = <&spdif1>;
-		spdif-out;
-		spdif-in;
-	};
 
-	sound-micfil {
-		compatible = "fsl,imx-audio-micfil";
-		model = "imx-audio-micfil";
-		cpu-dai = <&micfil>;
-	};
-*/
-};
-/* rfee
-&clk {
-	assigned-clocks = <&clk IMX8MM_AUDIO_PLL1>, <&clk IMX8MM_AUDIO_PLL2>;
-	assigned-clock-rates = <786432000>, <722534400>;
 };
 
-*/
 &iomuxc {
 	pinctrl-names = "default";
 
@@ -302,20 +240,6 @@
 			>;
 		};
 
-
-		/*rfee pinctrl_typec1: typec1grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_SD1_STROBE_GPIO2_IO11	0x159
-			>;
-		};
-
-		pinctrl_typec2: typec2grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_SD2_CD_B_GPIO2_IO12	0x159
-			>;
-		};*/
-
-
 		pinctrl_i2c4_gpio: i2c4grp-gpio {
 			fsl,pins = <
 				MX8MM_IOMUXC_I2C4_SCL_GPIO5_IO20        	0x1c3
@@ -329,85 +253,6 @@
 			>;
 		};
 
-
-/*rfee 
-		pinctrl_sai1: sai1grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_SAI1_MCLK_SAI1_MCLK	0xd6
-				MX8MM_IOMUXC_SAI1_TXFS_SAI1_TX_SYNC	0xd6
-				MX8MM_IOMUXC_SAI1_RXD7_SAI1_TX_SYNC	0xd6
-				MX8MM_IOMUXC_SAI1_TXC_SAI1_TX_BCLK	0xd6
-				MX8MM_IOMUXC_SAI1_TXD0_SAI1_TX_DATA0	0xd6
-				MX8MM_IOMUXC_SAI1_TXD1_SAI1_TX_DATA1	0xd6
-				MX8MM_IOMUXC_SAI1_TXD2_SAI1_TX_DATA2	0xd6
-				MX8MM_IOMUXC_SAI1_TXD3_SAI1_TX_DATA3	0xd6
-				MX8MM_IOMUXC_SAI1_TXD4_SAI1_TX_DATA4	0xd6
-				MX8MM_IOMUXC_SAI1_TXD5_SAI1_TX_DATA5	0xd6
-				MX8MM_IOMUXC_SAI1_TXD6_SAI1_TX_DATA6	0xd6
-				MX8MM_IOMUXC_SAI1_TXD7_SAI1_TX_DATA7	0xd6
-			>;
-		};
-
-		pinctrl_sai1_dsd: sai1grp_dsd {
-			fsl,pins = <
-				MX8MM_IOMUXC_SAI1_MCLK_SAI1_MCLK	0xd6
-				MX8MM_IOMUXC_SAI1_TXFS_SAI1_TX_SYNC	0xd6
-				MX8MM_IOMUXC_SAI1_RXD7_SAI1_TX_DATA4	0xd6
-				MX8MM_IOMUXC_SAI1_TXC_SAI1_TX_BCLK	0xd6
-				MX8MM_IOMUXC_SAI1_TXD0_SAI1_TX_DATA0	0xd6
-				MX8MM_IOMUXC_SAI1_TXD1_SAI1_TX_DATA1	0xd6
-				MX8MM_IOMUXC_SAI1_TXD2_SAI1_TX_DATA2	0xd6
-				MX8MM_IOMUXC_SAI1_TXD3_SAI1_TX_DATA3	0xd6
-				MX8MM_IOMUXC_SAI1_TXD4_SAI1_TX_DATA4	0xd6
-				MX8MM_IOMUXC_SAI1_TXD5_SAI1_TX_DATA5	0xd6
-				MX8MM_IOMUXC_SAI1_TXD6_SAI1_TX_DATA6	0xd6
-				MX8MM_IOMUXC_SAI1_TXD7_SAI1_TX_DATA7	0xd6
-			>;
-		};
-
-		pinctrl_sai3: sai3grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_SAI3_TXFS_SAI3_TX_SYNC     0xd6
-				MX8MM_IOMUXC_SAI3_TXC_SAI3_TX_BCLK      0xd6
-				MX8MM_IOMUXC_SAI3_MCLK_SAI3_MCLK        0xd6
-				MX8MM_IOMUXC_SAI3_TXD_SAI3_TX_DATA0     0xd6
-				MX8MM_IOMUXC_I2C4_SDA_GPIO5_IO21        0xd6
-			>;
-		};
-
-		pinctrl_sai5: sai5grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_SAI5_MCLK_SAI5_MCLK	0xd6
-				MX8MM_IOMUXC_SAI5_RXC_SAI5_RX_BCLK	0xd6
-				MX8MM_IOMUXC_SAI5_RXFS_SAI5_RX_SYNC	0xd6
-				MX8MM_IOMUXC_SAI5_RXD0_SAI5_RX_DATA0	0xd6
-				MX8MM_IOMUXC_SAI5_RXD1_SAI5_RX_DATA1    0xd6
-				MX8MM_IOMUXC_SAI5_RXD2_SAI5_RX_DATA2    0xd6
-				MX8MM_IOMUXC_SAI5_RXD3_SAI5_RX_DATA3    0xd6
-			>;
-		};
-
-		pinctrl_pdm: pdmgrp {
-			fsl,pins = <
-				MX8MM_IOMUXC_SAI5_MCLK_SAI5_MCLK	0xd6
-				MX8MM_IOMUXC_SAI5_RXC_PDM_CLK		0xd6
-				MX8MM_IOMUXC_SAI5_RXFS_SAI5_RX_SYNC	0xd6
-				MX8MM_IOMUXC_SAI5_RXD0_PDM_DATA0	0xd6
-				MX8MM_IOMUXC_SAI5_RXD1_PDM_DATA1	0xd6
-				MX8MM_IOMUXC_SAI5_RXD2_PDM_DATA2	0xd6
-				MX8MM_IOMUXC_SAI5_RXD3_PDM_DATA3	0xd6
-			>;
-		};
-
-		pinctrl_spdif1: spdif1grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_SPDIF_TX_SPDIF1_OUT	0xd6
-				MX8MM_IOMUXC_SPDIF_RX_SPDIF1_IN		0xd6
-			>;
-		};
-
-*/
-
 		pinctrl_uart1: uart1grp {
 			fsl,pins = <
 				MX8MM_IOMUXC_UART1_RXD_UART1_DCE_RX	0x140
@@ -809,56 +654,6 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_i2c3>;
 	status = "okay";
-
-/* rfee
-	pca6416: gpio@20 {
-		compatible = "ti,tca6416";
-		reg = <0x20>;
-		gpio-controller;
-		#gpio-cells = <2>;
-	};
-*/
-
-
-
-
-
-
-
-
-/* rfee 
-	ak4458_1: ak4458@10 {
-		compatible = "asahi-kasei,ak4458";
-		reg = <0x10>;
-		AVDD-supply = <&reg_audio_board>;
-		DVDD-supply = <&reg_audio_board>;
-	};
-
-	ak4458_2: ak4458@12 {
-		compatible = "asahi-kasei,ak4458";
-		reg = <0x12>;
-		AVDD-supply = <&reg_audio_board>;
-		DVDD-supply = <&reg_audio_board>;
-	};
-
-	ak5558: ak5558@13 {
-		compatible = "asahi-kasei,ak5558";
-		reg = <0x13>;
-		ak5558,pdn-gpio = <&pca6416 3 GPIO_ACTIVE_HIGH>;
-		AVDD-supply = <&reg_audio_board>;
-		DVDD-supply = <&reg_audio_board>;
-	};
-
-	ak4497: ak4497@11 {
-		compatible = "asahi-kasei,ak4497";
-		reg = <0x11>;
-		ak4497,pdn-gpio = <&pca6416 5 GPIO_ACTIVE_HIGH>;
-		AVDD-supply = <&reg_audio_board>;
-		DVDD-supply = <&reg_audio_board>;
-	};
-
-*/
-
 };
 
 
@@ -923,73 +718,6 @@
 	status = "okay";
 };
 
-/* rfee
-&sai1 {
-	pinctrl-names = "default", "dsd";
-	pinctrl-0 = <&pinctrl_sai1>;
-	pinctrl-1 = <&pinctrl_sai1_dsd>;
-	assigned-clocks = <&clk IMX8MM_CLK_SAI1_SRC>,
-			<&clk IMX8MM_CLK_SAI1_DIV>;
-	assigned-clock-parents = <&clk IMX8MM_AUDIO_PLL1_OUT>;
-	assigned-clock-rates = <0>, <49152000>;
-	clocks = <&clk IMX8MM_CLK_SAI1_IPG>, <&clk IMX8MM_CLK_DUMMY>,
-		<&clk IMX8MM_CLK_SAI1_ROOT>, <&clk IMX8MM_CLK_DUMMY>,
-		<&clk IMX8MM_CLK_DUMMY>, <&clk IMX8MM_AUDIO_PLL1_OUT>,
-		<&clk IMX8MM_AUDIO_PLL2_OUT>;
-	clock-names = "bus", "mclk0", "mclk1", "mclk2", "mclk3", "pll8k", "pll11k";
-	fsl,sai-multi-lane;
-	fsl,dataline,dsd = <0 0xff 0xff 2 0xff 0x11>;
-	dmas = <&sdma2 0 26 0>, <&sdma2 1 26 0>;
-	status = "okay";
-};
-
-&sai3 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_sai3>;
-	assigned-clocks = <&clk IMX8MM_CLK_SAI3_SRC>,
-			<&clk IMX8MM_CLK_SAI3_DIV>;
-	assigned-clock-parents = <&clk IMX8MM_AUDIO_PLL1_OUT>;
-	assigned-clock-rates = <0>, <24576000>;
-	status = "okay";
-};
-
-&sai5 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_sai5>;
-	assigned-clocks = <&clk IMX8MM_CLK_SAI5_SRC>,
-			<&clk IMX8MM_CLK_SAI5_DIV>;
-	assigned-clock-parents = <&clk IMX8MM_AUDIO_PLL1_OUT>;
-	assigned-clock-rates = <0>, <49152000>;
-	clocks = <&clk IMX8MM_CLK_SAI5_IPG>, <&clk IMX8MM_CLK_DUMMY>,
-		<&clk IMX8MM_CLK_SAI5_ROOT>, <&clk IMX8MM_CLK_DUMMY>,
-		<&clk IMX8MM_CLK_DUMMY>, <&clk IMX8MM_AUDIO_PLL1_OUT>,
-		<&clk IMX8MM_AUDIO_PLL2_OUT>;
-	clock-names = "bus", "mclk0", "mclk1", "mclk2", "mclk3", "pll8k", "pll11k";
-	fsl,sai-asynchronous;
-	status = "disabled";
-};
-
-&spdif1 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_spdif1>;
-	assigned-clocks = <&clk IMX8MM_CLK_SPDIF1_SRC>,
-			<&clk IMX8MM_CLK_SPDIF1_DIV>;
-	assigned-clock-parents = <&clk IMX8MM_AUDIO_PLL1_OUT>;
-	assigned-clock-rates = <0>, <24576000>;
-	clocks = <&clk IMX8MM_CLK_AUDIO_AHB_DIV>, <&clk IMX8MM_CLK_24M>,
-		<&clk IMX8MM_CLK_SPDIF1_DIV>, <&clk IMX8MM_CLK_DUMMY>,
-		<&clk IMX8MM_CLK_DUMMY>, <&clk IMX8MM_CLK_DUMMY>,
-		<&clk IMX8MM_CLK_AUDIO_AHB_DIV>, <&clk IMX8MM_CLK_DUMMY>,
-		<&clk IMX8MM_CLK_DUMMY>, <&clk IMX8MM_CLK_DUMMY>,
-		<&clk IMX8MM_AUDIO_PLL1_OUT>, <&clk IMX8MM_AUDIO_PLL2_OUT>;
-	clock-names = "core", "rxtx0", "rxtx1", "rxtx2", "rxtx3",
-		"rxtx4", "rxtx5", "rxtx6", "rxtx7", "spba", "pll8k", "pll11k";
-	status = "okay";
-};
-
-*/
-
-
 &fec1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_fec1>;
@@ -1046,23 +774,6 @@
 	status = "okay";
 };
 
-/* rfee &usbotg1 {
-	dr_mode = "otg";
-	extcon = <0>, <&typec1_ptn5110>;
-	picophy,pre-emp-curr-control = <3>;
-	picophy,dc-vol-level-adjust = <7>;
-	status = "okay";
-};
-
-&usbotg2 {
-	dr_mode = "otg";
-	extcon = <0>, <&typec2_ptn5110>;
-	picophy,pre-emp-curr-control = <3>;
-	picophy,dc-vol-level-adjust = <7>;
-	status = "disabled";
-};*/
-
-
 &usbotg1 {
 	vbus-supply = <&reg_usb_otg_vbus>;
 	pinctrl-names = "default";
@@ -1081,34 +792,6 @@
 	status = "okay";
 };
 
-
-/* rfee
-&usdhc1 {
-	pinctrl-names = "default", "state_100mhz", "state_200mhz";
-	pinctrl-0 = <&pinctrl_usdhc1>, <&pinctrl_usdhc1_gpio>;
-	pinctrl-1 = <&pinctrl_usdhc1_100mhz>, <&pinctrl_usdhc1_gpio>;
-	pinctrl-2 = <&pinctrl_usdhc1_200mhz>, <&pinctrl_usdhc1_gpio>;
-	bus-width = <4>;
-	vmmc-supply = <&reg_sd1_vmmc>;
-	pm-ignore-notify;
-	keep-power-in-suspend;
-	non-removable;
-	status = "okay";
-};
-*/
-
-/* rfee 
-&usdhc2 {
-	pinctrl-names = "default", "state_100mhz", "state_200mhz";
-	pinctrl-0 = <&pinctrl_usdhc2>, <&pinctrl_usdhc2_gpio>;
-	pinctrl-1 = <&pinctrl_usdhc2_100mhz>, <&pinctrl_usdhc2_gpio>;
-	pinctrl-2 = <&pinctrl_usdhc2_200mhz>, <&pinctrl_usdhc2_gpio>;
-	cd-gpios = <&gpio1 15 GPIO_ACTIVE_LOW>;
-	bus-width = <4>;
-	vmmc-supply = <&reg_usdhc2_vmmc>;
-	status = "okay";
-};*/
-
 /*rfee  from uboot*/
 &usdhc2 {
 	pinctrl-names = "default", "state_100mhz", "state_200mhz";
@@ -1158,13 +841,3 @@
 &vpu_h1 {
 	status = "okay";
 };
-/*rfee 
-&micfil {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_pdm>;
-	assigned-clocks = <&clk IMX8MM_CLK_PDM_SRC>, <&clk IMX8MM_CLK_PDM_DIV>;
-	assigned-clock-parents = <&clk IMX8MM_AUDIO_PLL1_OUT>;
-	assigned-clock-rates = <0>, <196608000>;
-	status = "okay";
-};
-*/
-- 
2.17.1

